src/main/java/core/mate/academy/model/ExcavatorProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {

```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the start of the class definition (line 5).